dlp/snippets/deid.py

```diff
@@ -937,6 +937,93 @@ def deidentify_with_replace_infotype(project, item, info_types):
 # [END dlp_deidentify_replace_infotype]
 
 
+# [START dlp_deidentify_exception_list]
+def deindentify_with_exception_list(
+    project,
+    content_string,
+    info_types,
+    exception_list
+):
+    """Uses the Data Loss Prevention API to deidentify sensitive data in a
+      string by ignore matches against custom list.
+
+    Args:
+        project: The Google Cloud project id to use as a parent resource.
+        content_string: The string to deidentify (will be treated as text).
+        info_types: A list of strings representing info types to look for.
+            A full list of info type categories can be fetched from the API.
+        exception_list: The list of strings to ignore matches on.
+
+    Returns:
+          None; the response from the API is printed to the terminal.
+    """
+    # Import the client library
+    import google.cloud.dlp
+
+    # Instantiate a client
+    dlp = google.cloud.dlp_v2.DlpServiceClient()
+
+    # Convert the project id into a full resource id.
+    parent = f"projects/{project}"
```

Comment:
move this down below just before you need it

Reply:
That's how with other code samples, but have moved this down where it is being used, in the next push. Thank you for reviewing.